freqtrade/persistence.py

```diff
@@ -184,8 +189,12 @@ class Trade(_DECL_BASE):
     open_order_id = Column(String)
     # absolute value of the stop loss
     stop_loss = Column(Float, nullable=True, default=0.0)
+    # percentage value of the stop loss
+    stop_loss_pct = Column(Float, nullable=True, default=0.0)
```

Comment:
I would not default these to 0, but to None.

it's less missleading as on the first iteration (before it hits stoploss) this will not be set.
and the column is nullable anyway, so it shouldn't hurt ... ?